tests_disabled_tcpseg.json

```diff
@@ -180,6 +176,34 @@
         {
             "name": "http2_general.test_h2_streams.TestH2Stream.test_max_concurrent_stream",
             "reason": "Disabled by test issue #525"
+        },
+	{
+	     "name": "ws.test_ws_ping.WsPing",
+             "reason": "These tests should not be run with TCP segmentation."
+        },
+	{
+	     "name": "ws.test_ws_ping.WssPing",
+             "reason": "These tests should not be run with TCP segmentation."
+        },
+	{
+	     "name": "ws.test_ws_ping.WssPingProxy",
+             "reason": "These tests should not be run with TCP segmentation."
+        },
+	{
+	     "name": "ws.test_ws_ping.CacheTest",
+             "reason": "These tests should not be run with TCP segmentation."
+        },
+	{
+	     "name": "ws.test_ws_ping.WssStress",
+             "reason": "These tests should not be run with TCP segmentation."
+        },
+	{
+	     "name": "ws.test_ws_ping.WsScheduler",
+             "reason": "These tests should not be run with TCP segmentation."
+        },
+	{
+	     "name": "ws.test_ws_ping.RestartOnUpgrade",
+             "reason": "These tests should not be run with TCP segmentation."
```

Comment:
It'd be good to describe somewhere why the tests aren't supposed to work with TCP segmentation